### ISSUE TITLE
The val dict and the test dict should be the different, isn't it? 

### ISSUE THREAD

**Issue by YijianLiu** [HRNet/HRNet-Object-Detection#31]
I see that in the configs, your val dataset and the test dataset are  same, but the val dataset involves training, I think they must be different.

**Comment by wondervictor** [comment]
>  but the val dataset involves training, I think they must be different
`coco-val2017` is the validation set and `coco-train2017` is the training set. **Neither of them involves the other**

**Comment by YijianLiu** [comment]
1

**Comment by YijianLiu** [comment]
> > but the val dataset involves training, I think they must be different
> 
> `coco-val2017` is the validation set and `coco-train2017` is the training set. **Neither of them involves the other**

Thanks so much, and whether you use coco-val2017 to test the latest epoch model?  The testing set and the validation set are the same dataset?

**Comment by wondervictor** [comment]
Our results are all evaluated on the model of the last epoch.
The testing set of COCO has been divided into two subsets (test-dev/test-challenge). If you want to know more about it, please see [cocodataset.org ](cocodataset.org )
